package-lock.json

```diff
@@ -1535,7 +1468,7 @@
 		},
 		"async": {
 			"version": "0.2.10",
-			"resolved": "https://registry.npmjs.org/async/-/async-0.2.10.tgz",
```

Comment:
I don't get this, I think I've seen npm change this from `http` to `https` before

Reply:
Bug and solution: https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/31